README.md

```diff
@@ -48,6 +48,7 @@ This appender uses  [LogzioSender](https://github.com/logzio/logzio-java-sender)
 | **addHostname**       | *false*                                    | Optional. If true, then a field named 'hostname' will be added holding the host name of the machine. If from some reason there's no defined hostname, this field won't be added |
 | **additionalFields**       | *None*                                    | Optional. Allows to add additional fields to the JSON message sent. The format is "fieldName1=fieldValue1;fieldName2=fieldValue2". You can optionally inject an environment variable value using the following format: "fieldName1=fieldValue1;fieldName2=$ENV_VAR_NAME". In that case, the environment variable should be the only value. In case the environment variable can't be resolved, the field will be omitted. |
```

Comment:
_You can optionally inject an environment variable or system property value using the following format: "fieldName1=fieldValue1;fieldName2=$ENV_VAR_NAME"._ 

Please add "system property" in to the "token" parameter description.